docs/extras/conv.md

```diff
@@ -0,0 +1,131 @@
+# The converter tactic mode
+
+Inside a tactic block, one can use the keyword `conv` to enter converter
```

Comment:
`conv` stands for "conversion" not "converter". I believe the terminology comes from HOL light.

Reply:
It goes back quite a bit further, at least to LCF[1]:

> LCF’s rewriting functions are called conversions

That's at least the origin of the old conversion monad.  The new one is implemented differently, but I think it is supposed to behave similarly.

[1] Larry Paulson, A higher-order implementation of rewriting, 1983. http://www.cl.cam.ac.uk/~lp15/papers/Reports/TR035-lcp-rewriting.pdf

Reply:
The [directory](https://github.com/leanprover/lean/tree/master/library/init/meta/converter) is called `converter` and [conv.lean](https://github.com/leanprover/lean/blob/d6d44a19947e2575b3fceed6d61167d258c661fb/library/init/meta/converter/conv.lean#L6) starts with: "Converter monad for building simplifiers."

Myself I have no opinion about this (of course) but it seems the boss disagrees with you.